CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+## Contributing
```

Comment:
The issue is about creating a basic readme file, I don't know if this file should be here. What do you think @jouderianjr @adrianobrito ? Maybe could be in another commit.

Reply:
I think that's a good idea keep this file in repository. It make easy to understand the necessary guidelines to external collaborators. It's fairly used pratice in majors projects like [Scala language](https://github.com/scala/scala/blob/2.12.x/CONTRIBUTING.md) and [Atom editor](https://github.com/atom/atom/blob/master/CONTRIBUTING.md).

However, It's necessary to discuss with the team members to ellaborate the content of this file.